library/src/main/java/com/schibsted/spain/barista/util/ViewTreeAnalyzer.java

```diff
@@ -0,0 +1,24 @@
+package com.schibsted.spain.barista.util;
+
+import android.view.View;
+import android.view.ViewGroup;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class ViewTreeAnalyzer {
+  public static List<View> getAllChildren(View parent) {
+    if (!(parent instanceof ViewGroup)) {
+      return Collections.singletonList(parent);
```

Comment:
Well, the thing is that the code is recursive. Actually, you will call this code to get all the childs of a View, but actually what it does is to return all non ViewGroup things below it. If you can suggest a better name for the method...